package.json

```diff
@@ -11,6 +11,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "@types/react-select": "^4.0.15",
```

Comment:
Is this `@types/react-select`  package a main dependency or a development dependency?

Reply:
I found the problem,It was installed during installation of react-select package. I found that it's not necessary  for our project. I am removing it .